client/battle/BattleWindow.cpp

```diff
@@ -51,7 +51,7 @@ BattleWindow::BattleWindow(BattleInterface & owner):
 
 	REGISTER_BUILDER("battleConsole", &BattleWindow::buildBattleConsole);
 	
-	const JsonNode config(ResourceID("config/widgets/BattleWindow2.json"));
+	const JsonNode config(ResourceID("config/widgets/BattleWindow.json"));

```

Comment:
I was thinking to limit vcmi-extras max compatibility by vcmi-1.3, so old mod won't be compatible with new version. But in the end decided to avoid this and limited compatibility only for extendedRmg submod https://github.com/vcmi-mods/vcmi-extras/commit/25dda0a591f93e2f8504f044b472576172407a01

Will revert